src/Apps/Order/Routes/Shipping2/Shipping2.jest.tsx

```diff
@@ -728,19 +737,23 @@ describe("Shipping", () => {
             )
 
             expect(
-              screen.queryByText("This field is required")
+              screen.queryByText(/[\w\s]is required/)
             ).not.toBeInTheDocument()
 
             await fillAddressForm(validAddress)
             userEvent.clear(screen.getByPlaceholderText("Street address"))
 
             await userEvent.click(screen.getByText("Save and Continue"))
 
-            expect(screen.getByText("This field is required")).toBeVisible()
-            expect(env.mock.getAllOperations()).toHaveLength(0)
+            await waitFor(() => {
+              expect(
+                screen.getByText("Street address is required")
+              ).toBeVisible()
+              expect(env.mock.getAllOperations()).toHaveLength(0)
+            })
```

Comment:
`await waitFor(() => somethingAsyncThatMightFail)` is a good first choice before `flushPromiseQueue`